app/jobs/recaptcha_annotate_job.rb

```diff
@@ -3,5 +3,6 @@
 class RecaptchaAnnotateJob < ApplicationJob
   def perform(assessment:)
     RecaptchaAnnotator.submit_assessment(assessment)
+    assessment.destroy
```

Comment:
Noting that these changes would only be effective for half of the worker instances in the 50/50 state, therefore some stale assessments could remain. That might be acceptable, since this is just a clean-up of unnecessary data, but the other option would be to split out a separate pull request with just these changes to deploy before the main pull request.

A third option could be to move the deletion into the `RecaptchaAnnotator` class. I wasn't originally a fan of that because it feels like the annotator's job should be to just submit the annotation, but maybe it fits the scope of "annotating" to include the clean-up of the assessment record. Pragmatically, it would also avoid having to deal with these issues or a third pull request.

Reply:
Actually, the third option might still be best, but it doesn't actually work around the 50/50 state, since the new application instances would _schedule_ the job from `RecaptchaAnnotator`, but the actual submission in `RecaptchaAnnotator` would still happen in a (possibly-old) worker instance.

Reply:
Based on a [conversation in Slack](https://gsa-tts.slack.com/archives/C05MGJ72GU9/p1740750351616549), I added timestamps to the table in be44d95, which should more easily allow us to delete any stale data in the future.